RLTSquare/TrainingTask/etc/di.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<!--
+/**
+* @author RLTSquare
+* @copyright Copyright (c) 2022 RLTSquare (https://www.rltsquare.com/)
+* @package RLTSquare_TrainingTask
+*/
+ -->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="../../../../../lib/internal/Magento/Framework/ObjectManager/etc/config.xsd">
+    <type name="RLTSquare\TrainingTask\Logger\Handler">
```

Comment:
Overcomplicated code - the code around setting custom file name / logger is unnecessary overcomplicated. See the https://drive.google.com/file/d/1ed3KeOMXJoXriWg7CJeEh4rEGzFkU1vk/view?usp=share_link . There is no need to write any separate logger/handler files.